books/precalculus/precalculus_recipe.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
```

Comment:
To be able to import this recipe into other files, the `.rb` extension is needed; this is why I moved it into another file